testsuite/tests/typing-layouts-unboxed-records/basics.ml

```diff
@@ -31,6 +31,17 @@ type t = #{ i : int; j : int; }
 val add : t -> int = <fun>
 |}]
 
+let bad_match (x : t) =
+  match x with
+  | _ -> .
+[%%expect{|
+Line 3, characters 4-5:
+3 |   | _ -> .
+        ^
+Error: This match case could not be refuted.
+       Here is an example of a value that would reach it: "{ _ }"
```

Comment:
This says `"{ _ }"` but I guess I expect it to say `#{ _ }`.  If it's hard to fix, just leave a CR, not worth spending a bunch of time investigating.